### PR TITLE
[wip] Api integration

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

File: geoportal/webpack.api.js.mako [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      ],
+      alias: {
+        api: '/usr/lib/node_modules/ngeo/api/src',
+      }
+    },
+    resolveLoader: {
+      modules: ['/usr/lib/node_modules'],
+    }
+  };
+};
```
Comment (sbrunner) [inline geoportal/webpack.api.js.mako]: Can be not interpreted by using `{{package}}` in the scaffold :-)

File: geoportal/demo_geoportal/static-ngeo/api/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import Map from 'api/Map.js';
+import * as constants from 'api/constants.js';
+
+constants.themesUrl = 'https://www.example.com'
```
Comment (sbrunner) [inline geoportal/demo_geoportal/static-ngeo/api/index.js]: Missing semicolon.

File: geoportal/webpack.api.js.mako (hunk 1 above)
Comment (fredj) [reply geoportal/webpack.api.js.mako]: done

**Comment by fredj** [comment]
https://github.com/camptocamp/c2cgeoportal/pull/4482

**Pull request closed without merging**